### ISSUE TITLE
Ozone PPB sensor output goes down when ozone PPB goes up in real life

### ISSUE THREAD

**Issue by OldNewbold** [ostaquet/Arduino-MQ131-driver#4]
Firstly, Thanks Ostaquet for your work on this library, it's the only one out there! But I have an issue.

Just put my winsen mq131 low concentration sensor in front of an ozone generator, was sitting at 12ppb, quickly dropped to 4ppb. Right now I have 2 such sensors facing directly into a powerful ozone generator blowing fairly high concentrations over the both of them. Other than sensor drift and noise (which understandably is not your fault!) I get on average lower readings for ozone.

Wiring is based it off the example provided, with the exception of 1M ohm load resistance trimpots which are set at 1M and 968k ohms respectively. I notice that the high concentration and low concentration sensors react oppositely to each other, i.e. low concentration resistance increases as ozone increases, whereas the high concentration resistance decreases as ozone increases. I will do some checks myself but perhaps there is something backwards?

**Comment by OldNewbold** [comment]
P.S. Ozone sensing seems to be dominated by untested products. Very Troubling.

**Comment by ostaquet** [comment]
Dear OIdnewbold,
I didn't have the chance to test the sensor in a lab environment. I tested it by using the sensor outside in clear air and compare with results from a local air quality station.

According to the specification, the _sensor conductivity gets lower along with the gas concentration rising_. It means that the resistance (Rs) increase while the gas concentration is higher. To be accurate, it's important that the R0 is correctly calibrated. In my experience, I saw crazy output if the preheat period of 48 hours is not respected.

So, we have Rs increasing. So, Rs/R0 is increasing and according to the sensitivity curves, it gives us the concentration based on the formula defined in the [Excel file](Arduino-MQ131-driver/datasheet/Sensitivity_curves.xlsx).

I honestly don't see why your sensor is working backwards... except if it's not a MQ131 (a lot of Winsen sensors, like the MQ7, are working on the other way: if gas concentration is increasing, the resistance is becoming lower).
